### DISCUSSION TITLE
Mobile Theme

### DISCUSSION THREAD

**Discussion by nicosea** [levito/tt-rss-feedly-theme#88] [Ideas]
Would be an idea (and possible) to have a feedly theme which looks also good on mobile phones?

**Comment by levito** [comment]
Unfortunately that's almost impossible to do. I'd suggest using the app.

**Comment by levito** [comment]
Hey @nicosea, I just made the theme much more mobile friendly! You can now
- swipe left/right between feeds holder and main content area, and – if you have the plugin `toggle_sidebar` installed – also open/close the feeds holder by tapping the left side of the screen
- always see the action icons and published date on headline rows (desktop: icons on hover, date only when not hovering)
- use all the views on mobile (three panel widescreen view will require the plugin `close_button` because headlines and detail view are always full width)
- use all the dialogs and even the preferences

Please feel free to give it a try and let me know how you like it!
